seonshine_frontend/src/routes/paths.ts

```diff
@@ -5,6 +5,10 @@ export const paths = {
   signUp: '/sign-up',
   test: '/test',
   pageNotFound: '/404',
-  main: '/main',
   admin: '/admin',
+  dashboard: '/dashboard',
+  userManagement: '/user-management',
```

Comment:
the user management has multiple paths. Please define as object,  Ex `user: {index: 'user', detail: ':id', register: 'register', edit: ':id/edit'}`